config-example.yaml

```
@@ -140,6 +140,23 @@ ephemeral_node_inactivity_timeout: 30m
 database:
   type: sqlite
 
+  # Enable debug mode. Set log.level to "debug".
```

It looks like some settings will require `trace` based on the wrapper

```suggestion
  # Enable debug mode. This setting requires the log.level to be set to "debug" or "trace".
```